src/AccountingAPIClient.ts

```diff
@@ -803,15 +805,15 @@ export class AccountingAPIClient extends BaseAPIClient {
 				return this.oauth1Client.post<TrackingCategoriesResponse>(endpoint, trackingOption);
 			},
 			delete: async (args: { TrackingCategoryID: string, TrackingOptionID: string }): Promise<any> => {
-				const endpoint = 'trackingcategories/' + args.TrackingCategoryID + '/Options/' + args.TrackingOptionID;
+				const endpoint = this.accountingBasePath + 'trackingcategories/' + args.TrackingCategoryID + '/Options/' + args.TrackingOptionID;
 				return this.oauth1Client.delete<any>(endpoint);
 			},
 		}
 	};
 
 	public users = {
 		get: async (args?: { UserID?: string } & HeaderArgs & QueryArgs): Promise<UsersResponse> => {
-			let endpoint = 'users';
+			let endpoint = this.accountingBasePath + 'users';
```

Comment:
There is a hulluva lot of `this.accountingBasePath` in this file :)

Could I suggest passing in an (optional) basePath to the BaseAPIClient constructor instead?